components/raftstore/src/coprocessor/config.rs

```diff
@@ -69,8 +69,10 @@ pub enum ConsistencyCheckMethod {
     Mvcc = 1,
 }
 
-/// Default region split size.
-pub const SPLIT_SIZE: ReadableSize = ReadableSize::mb(96);
+/// Default region split size. In version < 8.3.0, the default split size is
+/// 96MB. In version >= 8.3.0, the default split size is 256MB for the
+/// compatibility with the requirement for larger region size in TiKV.
```

Comment:
```suggestion
/// 96MB. In version >= 8.3.0, the default split size is increased to 256MB to
/// allow for larger region size in TiKV.
```

This might read better?

Reply:
thx